diff --git a/src/rbdmap b/src/rbdmap
index fa0cf7b821c..cd3f6661d97 100755
--- a/src/rbdmap
+++ b/src/rbdmap
@@ -83,10 +83,10 @@ do_map() {
 
 		if [ "$newrbd" ]; then
 			## Mount new rbd
-			MNT_RV=""
-			mount --fake /dev/rbd/$DEV >>/dev/null 2>&1 \
-			&& MNT_RV=$(mount -vn /dev/rbd/$DEV 2>&1)
-			[ -n "${MNT_RV}" ] && logger -p "daemon.debug" -t rbdmap "Mounted '${MAP_RV}' to '${MNT_RV}'"
+			# MNT_RV=""
+			# mount --fake /dev/rbd/$DEV >>/dev/null 2>&1 \
+			# && MNT_RV=$(mount -vn /dev/rbd/$DEV 2>&1)
+			# [ -n "${MNT_RV}" ] && logger -p "daemon.debug" -t rbdmap "Mounted '${MAP_RV}' to '${MNT_RV}'"
 
 			## post-mapping
 			if [ -x "/etc/ceph/rbd.d/${DEV}" ]; then
